tests/test_callbacks.py

```diff
@@ -0,0 +1,34 @@
+import os
+from dash_uploader import UploadStatus
+from app.app import disable_tabs
+from app.app import upload_data
+
+
+MOCK_FILE_PATH = os.path.join(os.path.dirname(__file__), "data/mock_obj_data.pkl")
```

Comment:
`"data/mock_obj_data.pkl"` will cause problems when running tests on Windows.

Better to create a `DATA_DIR` in the `__init__.py`, see [the example ](https://github.com/NPLinker/nplinker/blob/f767bfad74bc3c5e688864349c78060b4727c4d8/tests/unit/__init__.py#L4). 

Use `pathlib.Path` instead of `os.path` for path representation/operations.